test/base/test_store.py

```diff
@@ -404,31 +427,32 @@ def test_storage_objective_config():
             os.remove(fn)
 
 
-def test_result_from_hdf5_history():
+def test_result_from_hdf5_history(hdf5_file):
     problem = create_petab_problem()
-    with tempfile.TemporaryDirectory(dir=".") as tmpdirname:
```

Comment:
Note that there are issues on Windows with reopening temporary files, and `TemporaryDirectory` is therefore often preferable. Is pypesto tested on Windows?

Reply:
nope, there are currently not tests on windows. one could just rewrite the fixture as 
```python
with tempfile.TemporaryDirectory() as tmpdir:
    file = os.path.join(tmpdir, "file.hdf5")
    yield file
```
?

Reply:
True. Fine then.